examples/caltech101_cnn.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+
+from keras.datasets import caltech101
+from keras.preprocessing.image import ImageDataGenerator
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation, Flatten
+from keras.layers.normalization import LRN2D
+from keras.layers.convolutional import Convolution2D, MaxPooling2D, ZeroPadding2D
+from keras.optimizers import SGD
+from keras.utils import np_utils, generic_utils
+from six.moves import range
+
+import shutil
+import os
+from PIL import Image
+from resizeimage import resizeimage
+
+'''
+    Train a (fairly simple) deep CNN on the Caltech101 images dataset.
+    GPU run command:
+        THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatX=float32 python caltech101_cnn.py
+'''
```

Comment:
Could you provide a few more details, as many people probably aren't familiar with Caltech101. What is this about, why is it interesting, what accuracy does your model reach and how long does it take to train?